From 4208e62ebd1119c7b962915e0d118255daa5665f Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sat, 2 Nov 2024 22:25:37 +0800
Subject: [PATCH 134/257] AOSCOS: platform/x86: hp-wmi: Mark 8BAB board for
 OMEN thermal profile

Similar to 3a057bf30e044a51af8e6a8fe8cbfac49e2b9bc5 ("platform/x86:
hp-wmi: Add thermal profile support for 8BAD boards"), the HP OMEN 9 2023
(8BAB) board (China-specific model) should also be added to the list of
boards handled by the OMEN thermal profiles.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Link: https://t.me/c/1109254909/613200
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/platform/x86/hp/hp-wmi.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/platform/x86/hp/hp-wmi.c b/drivers/platform/x86/hp/hp-wmi.c
index db5fdee2109c..fcc0fdda0b38 100644
--- a/drivers/platform/x86/hp/hp-wmi.c
+++ b/drivers/platform/x86/hp/hp-wmi.c
@@ -68,7 +68,7 @@ static const char * const omen_thermal_profile_boards[] = {
 	"874A", "8603", "8604", "8748", "886B", "886C", "878A", "878B", "878C",
 	"88C8", "88CB", "8786", "8787", "8788", "88D1", "88D2", "88F4", "88FD",
 	"88F5", "88F6", "88F7", "88FE", "88FF", "8900", "8901", "8902", "8912",
-	"8917", "8918", "8949", "894A", "89EB", "8BAD", "8A42", "8A15"
+	"8917", "8918", "8949", "894A", "89EB", "8BAB", "8BAD", "8A42", "8A15"
 };
 
 /* DMI Board names of Omen laptops that are specifically set to be thermal
@@ -84,7 +84,7 @@ static const char * const omen_thermal_profile_force_v0_boards[] = {
  * "balanced" when reaching zero.
  */
 static const char * const omen_timed_thermal_profile_boards[] = {
-	"8BAD", "8A42", "8A15"
+	"8BAB", "8BAD", "8A42", "8A15"
 };
 
 /* DMI Board names of Victus 16-d1xxx laptops */
-- 
2.48.1

